airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -633,7 +634,7 @@ def get_databases(self, pattern='*'):
             return client.get_databases(pattern)
 
     def get_partitions(
-            self, schema, table_name, filter=None):
+            self, schema, table_name, filter=None):  # pylint: disable=redefined-builtin
```

Comment:
Should we fix it (will need note in UPDATING)? Or at least add TODO note.

Reply:
Fix